### ISSUE TITLE
Save to PDF doesn't work

### ISSUE THREAD

**Issue by Jont828** [atom-community/markdown-preview-plus#457]
When I try to save to PDF, it creates a plain text file with 9 bytes. I open the preview with `ctrl+shift+m` and press `ctrl+shift+s` to open up the save panel. Saving to HTML, opening the web page, and using the "save to pdf" option in Chrome's print menu seems to be a viable work around right now, though.

**Comment by lierdakil** [comment]
This might be related to your Electron version. Upstream only recently started working on migration to Electron 3.1. I honestly have no idea what your distro maintainers were thinking building Atom with Electron 4. Additionally, your Chrome and Node versions (at least as reported by `atom --version`) are way too recent too.

Honestly, all bets are off on an unsupported configuration like this. Try installing Atom from deb/rpm, or perhaps building from source and see if that works better?

FWIW, I can not reproduce this on
```
Atom    : 1.37.0-beta0
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

**Comment by lierdakil** [comment]
I *think* this should be fixed in v3.11.3. Well, at least I was able to reproduce the issue with Atom v1.41.0 (which uses Electron 4.2). This is apparently a bug in Electron 4, and I've added a workaround in MPP v3.11.3.

Anyway, please let me know if it's still reproducible post-v3.11.3. Thanks.
